spec/models/tag_substitutions_spec.rb

```diff
@@ -2,7 +2,7 @@
 
 require 'rails_helper'
 
-describe TagSubstitution do
+describe TagSubstitution, warren: true do
```

Comment:
what does the `warren: true` do, is that new since your recent work on warren?

Reply:
Nope, its actually been around for a little while. It enables logging of messages for testing purposes:
https://rubydoc.info/gems/sanger_warren/Warren/Handler/Test

Reply:
I'll make an issue to fix the markup on that documentation... its not supposed to insert whole iframes for the 'useful' messages section, and they'd be better off as simple links.